tests/integration/common/s2n_test_scenario.py

```diff
@@ -195,7 +232,7 @@ def run_scenarios(test_func, scenarios):
 
 
 def get_scenarios(host, start_port, s2n_modes=Mode.all(), versions=[None], ciphers=[None],
-                  curves=ALL_CURVES, certs=ALL_CERTS, s2n_flags=[], peer_flags=[]):
+                  curves=[None], certs=ALL_CERTS, s2n_flags=[], peer_flags=[]):
```

Comment:
I made this None, as the curves are determined by the libcrypto_version.

Reply:
Since you skip combos with curve==None, but your default is a curve list that just contains None, does that mean that by default get_scenarios produces no scenarios with this change? I think this needs to be Curve.all().